tool/build_check_deploy.sh

```diff
@@ -43,7 +43,7 @@ function check_formatting() {
 # Deploys website to Firebase, trying up to total_tries times.
 function deploy() {
   local total_tries="$1"
-  local remaining_tries=$(($total_tries - 1))
+  local remaining_tries=$(($total_tries))
```

Comment:
If we're dropping the arithmetic, then maybe just:

```bash
local remaining_tries=$total_tries
```

Reply:
Oh, duh.  Done.